src/training/validator.cpp

```diff
@@ -353,6 +355,12 @@ float TranslationValidator::validate(const std::vector<Ptr<ExpressionGraph>>& gr
 
   if(options_->hasAndNotEmpty("valid-translation-output")) {
     fileName = options_->get<std::string>("valid-translation-output");
+    fileName = fmt::format(fileName.c_str(),
+			   fmt::arg("E", state.epochs),
+			   fmt::arg("U", state.batches),
+			   fmt::arg("B", state.batchesEpoch),
+			   fmt::arg("W", state.labelsTotal), // alias for T
+			   fmt::arg("T", state.labelsTotal));
```

Comment:
Do we need both "W" and "T"? One of them is not in the help message.

Reply:
"W" removed.